.travis.yml

```diff
@@ -59,6 +59,7 @@ before_install:
 install:
   # Removing this here protects from inadvertent caching
   - rm -rf "$HOME/.m2/repository/org/apache/beam"
+  - rm -rf "$HOME/.m2/repository/org/apache/gearpump"
```

Comment:
Just noting that you could also do this by removing the `cache:` section. But I think you should keep it, because it saves a lot of traffic to Maven Central - it is mostly about the number of requests and the fact that they often fail.